kubemarine/system.py

```diff
@@ -265,6 +265,10 @@ def patch_systemd_service(group: DeferredGroup, service_name: str, patch_source:
     group.put(io.StringIO(utils.read_internal(patch_source)),
               f"/etc/systemd/system/{service_name}.service.d/{service_name}.conf",
               sudo=True)
+    if group.get_nodes_os() in ['rhel', 'rhel8', 'rhel9']:
+        group.sudo(f"chcon -u system_u -r object_r -t systemd_unit_file_t /etc/systemd/system/{service_name}.service.d")
+        group.sudo(f"chcon -u system_u -r object_r -t systemd_unit_file_t /etc/systemd/system/{service_name}.service.d/{service_name}.conf")
```

Comment:
Hi @krang404 , I see you have lost some my changes. I reworked this explicit chcon invocation to `mv -Z` option when copying the files on nodes. Could you revert the changes and try to rebase again?

Reply:
Oh, sorry. But if it not changed in my local repo installation in OL 9.2 and Rocky 9.2 are failed. I revert this changes. But are you sure that's ok?

Reply:
Could you revert all the changes you've made half an hour ago :)
The branch previously contained no `chcon` at all, but contained some other changes to support SELinux

Reply:
Now its clean from 'chcon'. And what changes i`ve lost?

Reply:
All the changes I have done in the branch approximately a week ago. You probably forgot to pull the changes, and discarded them using `--force`. It seems I had to repeat them. Could you add me as collaborator to your project?

Reply:
Ops. Shame on me. Invite you to collaboration in my fork. Double sorry.

Reply:
Updated 5f62bb60ce20b36c6796d012fbd89672d3083f09